SentryTestUtils/TestConstants.swift

```diff
@@ -0,0 +1,28 @@
+public struct TestConstants {
+    
+    /**
+     * Real dsn for integration tests.
+     */
+    public static let realDSN: String = "https://6cc9bae94def43cab8444a99e0031c28@o447951.ingest.sentry.io/5428557"
+    
+    public static func dsnAsString(username: String) -> String {
+        return "https://\(username):password@app.getsentry.com/12345"
+    }
+    
+    public static func dsn(username: String) throws -> SentryDsn {
+        return try SentryDsn(string: self.dsnAsString(username: username))
+    }
```

Comment:
I changed this function to throw the exception up the stack instead of calling `XCTFail` in a `do/catch`, since we can't import XCTest into a static lib target. That had some downstream effects that i'll call out in comments.